readme.md

```diff
@@ -101,21 +104,26 @@ _NB: Beyond session 1, these sessions aren't in any particular order. There's so
     - [GynePunk, the cyborg witches of DIY gynecology](http://www.makery.info/en/2016/06/30/gynepunk-les-sorcieres-cyborg-de-la-gynecologie-diy/). Ewen Chardronnet.
     - [What Was Life? Answers from Three Limit Biologies](https://anthropology.mit.edu/sites/default/files/documents/helmreich_what_was_life.pdf). Stefan Helmreich.
     - [Antiviral](https://en.wikipedia.org/wiki/Antiviral_(film)). Brandon Cronenberg.
+    
+- Session 10: Sapir-Whorf: Language & Perception
+    - [Shaka, When the Walls Fell](http://www.theatlantic.com/entertainment/archive/2014/06/star-trek-tng-and-the-limits-of-language-shaka-when-the-walls-fell/372107/). Ian Bogost. (watch the episode too)
+    - [Walls of the Tongue: A Sociolinguistic Analysis of Ursula K. Le Guin's _The Dispossessed_](http://linguistics.berkeley.edu/~dwbruhn/dwbruhn_376_Dispossessed.pdf). Daniel W. Bruhn.
+    - [Welcome to the Future Nauseous](http://www.ribbonfarm.com/2012/05/09/welcome-to-the-future-nauseous/). Venkatesh Rao.
```

Comment:
An essay about how the "future" is perceived, how the "present" is always felt to be normal, and how this prevents us from seeing the ways in which the world has changed drastically.